README.md

```diff
@@ -52,6 +53,8 @@ The autocomplete feature is only available in the main COBOL file, not in copybo
 ### Syntax and Semantic Check for Code
 This feature checks for mistakes and errors in COBOL code. The syntax check feature reviews the whole content of the code and suggests fixes, through syntax and semantic analysis which returns diagnostics on the entire context of the code, not just keywords.
 
+This feature is also enabled for CICS keywords inside `EXEC CICS` statements.
```

Comment:
Here would be better "CICS keywords and variables"

Reply:
changed in latest commit